src/cli/list.c

```diff
@@ -25,7 +25,6 @@ const struct argp list_usage = {
                 {"no-persistenced", 0x84, NULL, 0, "Don't include the NVIDIA persistenced socket", -1},
                 {"no-fabricmanager", 0x85, NULL, 0, "Don't include the NVIDIA fabricmanager socket", -1},
                 {"no-gsp-firmware", 0x86, NULL, 0, "Don't include GSP Firmware", -1},
-                {"no-cntlibs", 0x87, NULL, 0, "Don't overwrite host mounts with CUDA compat libs from the container", -1},
```

Comment:
Why can we just remove this without deprecating it? Do we also want to add a cuda-compat-mode flag here?

Reply:
It was erroneously added to mirror the `--no-cntlibs` option in the `configure` command. Note that the `list` command does NOT operate on a container and as such this option never had any effect since there are no container libraries.